x-pack/plugins/security_solution/public/detection_engine/rule_creation_ui/components/threatmatch_input/index.tsx

```diff
@@ -28,7 +28,7 @@ const CommonUseField = getUseField({ component: Field });
 
 interface ThreatMatchInputProps {
   threatMapping: FieldHook;
-  threatBrowserFields: Readonly<Record<string, Partial<BrowserField>>>;
+  threatBrowserFields: Readonly<BrowserFields>;
```

Comment:
This parameter is now unused in this component (since it also uses `QueryBarDefineRule`, which was modified above). You can similarly delete these references, which should light up the (also unused) props being passed to this component in the `step_define_rule` component.

Reply:
done! thanks